odk1-src/collect-forms.rst

```diff
@@ -203,7 +203,12 @@ Pulling Forms into Briefcase
 Deleting Forms
 ===============
 
-You can delete :formstate:`Blank` forms as well as filled forms in any state (:formstate:`Saved`, :formstate:`Finalized`, or :formstate:`Sent`). Deleting :formstate:`Blank` forms or filled forms in :formstate:`Saved` or :formstate:`Finalized` state removes every trace of that form definition or filled form. Deleting a :formstate:`Sent` form deletes the form contents but metadata associated with it including the deletion date and the instance name are maintained for display in the :guilabel:`View Sent Form` list.
+.. warning::
+    In Collect 1.28 filled instances of forms will still be editable after their blank form is deleted. This means that the form definition and media files will remain on the device until all the filled instances have been deleted.
```

Comment:
I think the warning here is actually about versions prior to v1.28 so I'd put that first. Then I think Collect v1.28+ is clearer. Or "Collect v1.28 and later".

Reply:
Also note we use a `v` prefix consistently when referring to software versions.